docs/howtoguides/how_to_roll_out_across_multiple_machines.rst

```diff
@@ -0,0 +1,48 @@
+.. _roll_out_multiple_machines:
+
+How to roll out your pro token across multiple machines
+=======================================================
+
+This guide will show you how to roll out your pro token across multiple machines using cloud-init for configuration.
+
+Configure using Cloud-init
+--------------------------
+
+The next step is to usea cloud-init to configure your machines with the pro token.
+
+To do this, a cloud-init configuratioin file is required. Create a cloud-init.yaml file with the following content:
+
+.. code-block:: yaml
+
+    #cloud-config
+    ubuntu_pro:
+        token: <YOUR_PRO_TOKEN>
+
+
+Replace <YOUR_PRO_TOKEN> with the pro token you received from the Ubuntu Pro Dashboard.
+
+For more information on how to configure the pro token with cloud-init, take a look at the guide here: `Configure Pro in cloud-init <CLOUD_INIT_PRO_>`_.
+
+Cloud-init also has comprehensive guides on configuring cloud-init for different platforms:
```

Comment:
This is step 3 - launching the machine with the configuration we want to apply. Since this can vary platform by platform I think it makes sense to provide multiple links as you have done. If we want to restrict these instructions to VMs/containers, we may also want to add Multipass and WSL.